### ISSUE TITLE
Scalars not an OutputArray

### ISSUE THREAD

**Issue by JonBoyleCoding** [twistedfall/opencv-rust#312]
I'm trying to use `mean_std_dev`, however whenever I've used it before in C++ I've used it with the outputs being Scalars. I'm using crate version 0.61.2. It notes in the docs:

> src: input array that should have from 1 to 4 channels so that the results can be stored in Scalar_ 's

When trying to provide these into `mean_std_dev` it comes up with the following:

```
error[E0277]: the trait bound `VecN<f64, 4_usize>: _OutputArrayTrait` is not satisfied
  --> src/pad_detector.rs:45:51
   |
45 |         opencv::core::mean_std_dev(&features_row, &mut mean, &mut std_dev, &no_array())?;
   |                                                   ^^^^^^^^^ the trait `_OutputArrayTrait` is not implemented for `VecN<f64, 4_usize>`
   |
   = note: required because of the requirements on the impl of `ToOutputArray` for `VecN<f64, 4_usize>`
   = note: required for the cast to the object type `dyn ToOutputArray`
```

Am I doing something wrong, or is this a current limitation of the library? Is there a way around this?

Thanks

**Comment by twistedfall** [comment]
Thanks for the report, I’ll check it out. For now the workaround would be to use `Mat` As output and extract the values into a `Scalar` after the operation.

**Comment by JonBoyleCoding** [comment]
Ok, thanks! I presume I'd need to make the Mat to have a size 4 due to the size of a Scalar?

**Comment by twistedfall** [comment]
Exactly

**Comment by twistedfall** [comment]
It's now fixed in v0.61.3
